lxd/storage/drivers/generic_vfs.go

```diff
@@ -170,7 +170,12 @@ func genericVFSMigrateVolume(d Driver, s *state.State, vol Volume, conn io.ReadW
 		path := shared.AddSlash(mountPath)
 
 		d.Logger().Debug("Sending filesystem volume", log.Ctx{"volName": vol.name, "path": path, "bwlimit": bwlimit, "rsyncArgs": rsyncArgs})
-		return rsync.Send(vol.name, path, conn, wrapper, volSrcArgs.MigrationType.Features, bwlimit, s.OS.ExecPath, rsyncArgs...)
+		err := rsync.Send(vol.name, path, conn, wrapper, volSrcArgs.MigrationType.Features, bwlimit, s.OS.ExecPath, rsyncArgs...)
+		if volSrcArgs.AllowInconsistent && strings.Contains(err.Error(), "exit status 24") {
```

Comment:
You should avoid testing string values for exit statuses as these can change based on the locale.
Instead you should be able to use the ExitStatus function to extract the status code from the error.

Reply:
Done.